resources/jupyter_notebook_config.partial.py

```diff
@@ -1,12 +1,3 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
-# Configuration file for ipython-notebook.
-
-c = get_config()
-c.NotebookApp.ip = '*'
-c.NotebookApp.open_browser = False
-c.NotebookApp.port = 8888
 
```

Comment:
The jupyter/minimal-notebook stack has all this and more: https://github.com/jupyter/docker-stacks/blob/master/minimal-notebook/jupyter_notebook_config.py